### ISSUE TITLE
SQLALchemy DBAPI: unexpected keyword argument 'username'

### ISSUE THREAD

**Issue by alexjironkin** [cloudera/impyla#343]
I have a Flask service with a bind to Impala:

```python
SQLALCHEMY_BINDS = {
  'datalake': 'impala://{}:{}@{}:21050/{}?use_ssl=true&auth_mechanism=PLAIN'.format(os.environ['DATALAKE_USER'], os.environ['DATALAKE_PASSWORD'], os.environ['DATALAKE_URI'], os.environ['DATALAKE_DB'] )
}
```

When a new session is created you get the following error:

>  TypeError("connect() got an unexpected keyword argument 'username'",)

As soon as I add a kwarg to `impala.dbapi.connect()` method and 
```python
if username is not None:
    user = username
```

This problem goes away. This is odd because I checked in PEP-249 for details and [footnotes](https://www.python.org/dev/peps/pep-0249/#footnotes) suggest implementing `user` as in Impala.

Is this a bug? Is there a better fix/workaround?

impyla: 0.14.1
sqlalchemy: 1.2.14

**Comment by timarmstrong** [comment]
Maybe you're running into the bug fixed by https://github.com/cloudera/impyla/pull/261/commits/7f52f88c9562e06b5aef4a11ea8691ed923be7b0?

**Comment by timarmstrong** [comment]
I just merged the fix last night so it's not in impyla 0.14.1

**Comment by alexjironkin** [comment]
👍 that looks exactly it! When were you thinking of cutting new release?

**Comment by timarmstrong** [comment]
@dknupp is looking at it as we speak.

**Comment by timarmstrong** [comment]
Fixed by  7f52f88

**Comment by dknupp** [comment]
@alexjironkin -- alpha release 0.15a1 is now live on pypi for you to try out.
` $ pip install impyla==0.15a1`
